main.py

```diff
@@ -25,6 +25,10 @@
 attachment_directory_filepath = os.environ.get("BUSTY_ATTACHMENT_DIR", "attachments")
 # The Discord role needed to perform bot commands
 dj_role_name = os.environ.get("BUSTY_DJ_ROLE", "bangermeister")
+# Max number of characters in an embed description (currently 4096 in Discord)
+embed_description_limit = 4096
+# Color of !list embed
+list_embed_color = 0xDD2E44
```

Comment:
I'd separate these out from the settings section. Also, as they're constants, canonically they'd be in `UPPER_SNAKE_CASE`.